tdrs-frontend/.env.example

```diff
@@ -27,5 +28,7 @@ REACT_APP_EVENT_THROTTLE_TIME=60000
 # Presently both of these are possible (in an .scss file):
 #  @import 'theme/_global.scss';
 #  @import '../../theme/_global.scss';
-# Without the variable, only the relavtive import is possible
+# Without the variable, only the relative import is possible
 SASS_PATH=node_modules:src
+
+PUBLIC_URL=url
```

Comment:
@carltonsmith @jtwillis92 Was one of our conclusions from Dev Sync that we don't want this line?

Reply:
Actually, it looks like this only gets used in production for service workers (see `/tdrs-frontend/src/serviceWorker.js`). We probably don't need it for local dev but we will for production.

Reply:
@jtwillis92 literal production or any deployed environment?

Reply:
@carltonsmith any environment where the `NODE_ENV` variable is set to `production`. The `PUBLIC_URL` variable is used by code generated with `create-react-app` (https://blog.bitsrc.io/using-service-workers-with-react-27a4c5e2d1a9)

Reply:
That makes sense, in that case I agree we should take it out of the local env example file. And FWIW my app runs fine locally without it. 😄